### DISCUSSION TITLE
vcpkg edit: run editor detached

### DISCUSSION THREAD

**Discussion by Mixaill** [microsoft/vcpkg#17723] [Ideas]
It would be nice if `vcpkg edit <package_name>` will be executed without command line locking

![image](https://cloud.githubusercontent.com/assets/704382/23837040/b890f390-0792-11e7-9c96-8def26d583e0.png)

**Comment by ras0219-msft** [comment]
One thing to note when implementing this feature is that many editors _need_ to lock the command line (such as if you prefer to use vim like `git.exe` does).

**Comment by alexkaratarakis** [comment]
If you using VS Code, a workaround is to have (at least) one instance of VS Code open.
That way, `vcpkg edit` will launch VS Code but won't lock the console.

**Comment by MVoz** [comment]
`set EDITOR=C:\Program Files (x86)\Notepad++\notepad++.exe`

либо другой

`set EDITOR=notepad.exe`
